pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+func updateEndpointAccessCmd(cmd *cmdutils.Cmd) {
```

Comment:
based on current convention this should be called `<action>Cmd`, where `<action>` is a camel case version of the name of the command

Reply:
changed.